actions/secret/action.yml

```diff
@@ -1,5 +1,5 @@
 name: 'ggshield scan'
-description: 'Scan commits for hardcoded secrets and security policy breaks.'
+description: 'Scan commits for hardcoded secrets.'
```

Comment:
Shouldn't we remove completely everything about the old and deprecated `ggshield scan` commands?

Reply:
The "ggshield scan" name here refers to the name of the GitHub action, not the executed ggshield command (which is `ggshield secret scan`). But it should be safe to rename the action to "ggshield secret scan".